server/helpers/dateHelper/index.js

```diff
@@ -0,0 +1,29 @@
+const moment = require("moment");
+const boom = require("boom");
+
+/**
+ * returns the date of one week/two weeks/ or three weeks from our current date.
+ * @param {integer} numberOfWeeks number of weeks to add to current date.
+ * @returns {date} date object representing the target date based on number of weeks.
+ */
+
+const getTargetDate = (numberOfWeeks) => {
+  // input validation
+  if (typeof numberOfWeeks !== "number") {
+    throw boom.badData("dateHelper/index.js: number of weeks must be a number");
+  }
+  if (numberOfWeeks > 3 || numberOfWeeks < 1) {
+    throw boom.badData("dateHelper/index.js: number of weeks must be an integer between 1 and 3");
+  }
+
+  // trims the curent date to only show the year, month and day.
+  // allows us to perform the check from midnight to avoid including one more or one less day.
+  const currentDate = moment().format().toString().substring(0, 10); // eg: 2019-08-22
```

Comment:
```suggestion
  const currentDate = moment().startOf("day");
```
this will give you something like `2019-09-01T00:00:00.000Z`
I am not sure what the Z implies but this sets the date to the start of the day
[check this](https://momentjs.com/docs/#/manipulating/start-of/)

or if you just want its output as `2019-08-22` you could say 
```js
const currentDate = moment().format("YYYY-MM-DD").toString();
```